src/internal/internalAPIs.ts

```diff
@@ -62,6 +63,13 @@ function handleChangeSettings(): void {
   }
 }
 
+function handleGetLogRequest(): void {
```

Comment:
We can move all functions related to log into its separate namespace/file similar to menus.ts